pkg/kine/logstructured/sqllog/otel.go

```diff
@@ -0,0 +1,20 @@
+package sqllog
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/metric"
+)
+
+const name = "sqllog"
+
+var (
+	tracer     = otel.Tracer(name)
+	meter      = otel.Meter(name)
+	compactCnt metric.Int64Counter
+)
+
+func init() {
+	compactCnt, _ = meter.Int64Counter(fmt.Sprintf("%s.compact", name), metric.WithDescription("Number of compact requests"))
```

Comment:
What happens if there is an error here? I think that `compactCnt` will be `nil` and so the code will panic on its first usage. I think we should panic here, if an error occurs, instead of having an error down the line.

Reply:
I'm adding it for completeness. FYI: Otel has this noop implementation which will just do nothing if something isn't defined: https://opentelemetry.io/docs/specs/otel/metrics/noop/#counter-add